.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -476,6 +484,10 @@ stages:
           baseImage: alpine
           artifactSuffix: linux-musl-x64
           useNativeSdkVersion: ""
+        x64-r2r:
+          baseImage: centos7
+          artifactSuffix: linux-x64-r2r
+          useNativeSdkVersion: true
```

Comment:
The profiler doesn't build managed libs, so I don't think this is necessary? 🤔

Reply:
Removed!